README.md

```diff
@@ -490,13 +490,14 @@ You first need to create an [Atlas account](https://atlas.hashicorp.com/account/
 * **filepath**: Conditionally Optional, Path to cloudformation template file.
 * **stack_name**: CloudFormation Stack Name.
 * **stack_name_prefix**: Optional, CloudFormation Stack Name Prefix.
-* **promote**: Boolean (default `false`), if true changes are deployed otherwise change set is created.
+* **promote**: Optional, Boolean (default `true`), if true changes are deployed otherwise change set is created.
 * **role_arn**: Optional, AWS Role ARN (assumed by cloudformation).
 * **sts_assume_role**: Optional, AWS Role ARN for cross account deployments (assumed by travis using given AWS credentials).
 * **capabilities**: Optional, Cloudformation allowed capabilities (allowed values: `CAPABILITY_IAM`, `CAPABILITY_NAMED_IAM`, `CAPABILITY_AUTO_EXPAND`; see [official documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html) for more information).
 * **wait**: Boolean (default `true`), should Travis wait to for cloudformation to complete stack creation/update. If true, cloudformation stack events will be streamed and displayed in logs.
 * **wait_timeout**: Optional (default `3600` seconds), How many seconds should we wait for stack creation/update.
```

Comment:
I will consider 50 minutes based on [Build Timeouts](https://docs.travis-ci.com/user/customizing-the-build#build-timeouts)

Reply:
Good point!